src/components/extension-landing/project-card.jsx

```diff
@@ -5,6 +5,7 @@ const ProjectCard = props => (
     <a
         className="project-card"
         href={props.cardUrl}
+        rel="noopener noreferrer"
```

Comment:
I don't think this is necessary in the case of internal links. Did eslint ask you to do this because of the `target="_blank"`?

Reply:
Yes, because of the target="_blank".

Is it easier/better to 
1) disable the rule in these cases; or
2) just use rel="noopener noreferrer" anyway? Does it hurt anything?

Reply:
It doesn't hurt anything unless it's to another of our internal html pages. In that case it would hide information from our own analytics, while not doing anything security-wise.  If they are internal links to PDFs, I don't think there's any reason not to.

So probably 1 in the case of internal links to html pages, 2 for PDFs.

According to https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md, this rule would not be triggered if it could tell that the link was internal/relative. There's an option to never enforce it if the link is dynamic, but I don't think there's any guarantee that we wouldn't supply a dynamic external link.